component-descriptor-v2-schema.yaml

```diff
@@ -91,6 +90,17 @@ definitions:
           - $ref: '#/definitions/githubAccess'
           - $ref: '#/definitions/httpAccess'
 
+  srcRef:
+    type: 'object'
+    description: 'a reference to a (component-local) source'
+    properties:
+      name:
```

Comment:
if we only use the name attribute to select a source, then we should get rid of the identity in sources. Otherwise we could have clashes.

Reply:
good point - will add `extraAttributes`, as well

Reply:
how about having an identity selector e.g.

```
srsRefs:
- identitySelector:
    name: my-name
    my-otherid: val
  labels: []
```

Reply:
in general, I like the idea of putting this into a separate attr. however, by doing that this would be inconsistent w/ the schema used in `resource`, `source`, and `component-reference` :-/

Reply:
I don't think that this would be inconsistent.
In the other resources the name attribute describes the item itself. In this context it is a reference. 
And I think name as toplevel attr. semms unclear that it is a reference not the name of the srcRef.

Having a dedicated identitySelector field makes it clear that this describes a reference to sources.

Reply:
would we require an individual selector to always match exactly one resource? from its name, we might e.g. support wildcards, or incomplete specifications (e.g. omit version, even if this would then match multiple resources)

Reply:
I changed it to `identitySelector`, assuming we will not require them to be unique

Reply:
I thinks its also valid to not be unique.
So I think its totally valid to multiple sources for a resource